tests/test_eradiate/_system/test_onedim_ckd_vs_mono.py

```diff
@@ -212,8 +212,7 @@ def test_550(reflectance, artefact_dir):
         "H2O-spectra-18300_18400",
     ]:
         test_tools.skipif_data_not_found(
-            "absorption_spectrum",
-            dataset_id,
+            f"spectra/absorption/{dataset_id}.nc",
```

Comment:
Shouldn't that be `f"spectra/absorption/us76_u86_4/{dataset_id}.nc"`?

Reply:
Most probably, this is typically a part of the code I can't verify since I don't have the data.

Reply:
Ah right, well, we won't upload these datasets, they're too big (or perhaps a subset for testing?)

Reply:
Yes, the new data workflow would make this possible. If those data are small enough, we might even consider shipping them through the data submodule.

Reply:
OK, I will look into that then